instrumentation/camel-2.20/javaagent/build.gradle.kts

```diff
@@ -45,9 +45,15 @@ dependencies {
   testImplementation("org.spockframework:spock-spring")
   testImplementation("javax.xml.bind:jaxb-api:2.3.1")
   testImplementation("org.elasticmq:elasticmq-rest-sqs_2.12:1.0.0")
+  testImplementation("com.squareup.okhttp3:okhttp:4.11.0")
 
   testImplementation("org.testcontainers:localstack")
-  testImplementation("org.testcontainers:cassandra")
+  testImplementation("org.testcontainers:cassandra:1.18.3")
```

Comment:
why do you need to set testcontainers version to `1.18.3`, isn't it automatically set because of https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/029e187281c19cc5a47f3a7b1e02985d0a45f94b/dependencyManagement/build.gradle.kts#L39

Reply:
I wasn't aware of that file and the justification for omitting them. Thank you for sharing, I have removed them.